Utilities/CodeGeneration/EdFi.Ods.CodeGen.Tests/Approval/5.2.0/DataStandard_520_ApprovalTests.Verify.Extensions.Sample.1.0.0_Std_5.2.0_Models_Entities_Entities.generated.approved.cs

```diff
@@ -536,8 +553,10 @@ public BusRoute()
         //                         Primary Key
         // -------------------------------------------------------------
         [DomainSignature]
+        [Key(6)]
```

Comment:
Aggregate root entities start their key values at `6` due to inherited properties that are already marked for serialization.